readme.md

```diff
@@ -41,6 +41,11 @@ You might also like [awesome-nodejs](https://github.com/sindresorhus/awesome-nod
 - [localForage-observable](https://github.com/thgreasi/localForage-observable) - Observables for localForage.
 
 
+## Libraries
+
```

Comment:
Can you add a note that these use Observables, but not necessarily follow the Observable spec?